code/game/turfs/simulated/minerals.dm

```diff
@@ -82,7 +82,11 @@
 	ScrapeAway(null, flags)
 	addtimer(CALLBACK(src, .proc/AfterChange), 1, TIMER_UNIQUE)
 	playsound(src, 'sound/effects/break_stone.ogg', 50, 1) //beautiful destruction
-
+	if(iscarbon(user)) 	//yogs - rock and stone
+		var/mob/living/carbon/C = user
+		if(prob(0.5) || (prob(1.5) && C.dna?.check_mutation(DWARFISM)))
```

Comment:
```suggestion
		if(prob(1.5) && C.dna?.check_mutation(DWARFISM)))
```

does this even work as it is currently for non dwarfs bc it doesnt look like it would